libs/api/domains/health-insurance/src/lib/soap/healthInsuranceAPI.ts

```diff
@@ -165,6 +164,8 @@ export class HealthInsuranceAPI {
         fyrrautgafulandkodi: inputObj.previousCountryCode,
         fyrriutgafustofnunlands: inputObj.previousIssuingInstitution ?? '',
         tryggdurfyrralandi: inputObj.isHealthInsuredInPreviousCountry,
+        tryggingaretturfyrralandi:
+          inputObj.hasHealthInsuranceRightInPreviousCountry,
```

Comment:
Shouldn't this module be in @island.is/health-insurance?

Reply:
This module is in @island.is/health-insurance. Same as above. Will be clean up